aten/src/ATen/templates/RegisterDispatchKey.cpp

```diff
@@ -57,9 +57,9 @@ namespace {
 
 ${dispatch_anonymous_definitions}
 
-TORCH_LIBRARY_IMPL(aten, ${DispatchKey}, m) {
-  ${dispatch_registrations}
-}
+${dispatch_registrations_begin} // TORCH_LIBRARY_IMPL(aten, ${DispatchKey}, m) {
+${dispatch_registrations}
+${dispatch_registrations_end} // }
```

Comment:
can we get away with leaving these in and just passing in an empty list of registrations?

Reply:
Are you suggesting that we could leave something like:
```
TORCH_LIBRARY_IMPL(aten, CPU, m) {

}
```
I'm not sure what's the underlying effect though

Reply:
Also @bdhirsh can you please help tag whoever knows how to test this in static dispatch use case?

Reply:
yep. That should be a no-op, so equivalent. And it makes the codegen a little cleaner, since we don't have to add as much special handling across the file for static dispatch.

Reply:
> can you please help tag whoever knows how to test this in static dispatch use case?

Right. Hey @swolchok, the mobile team is planning on making a few tweaks to static dispatch, that we don't expect to impact the server use case. Do you think we can rely on the existing CI (and internal tests, if there are any) to catch issues? Mostly just wondering if you think it's worth running any manual internal tests.

This specific PR updates the static dispatch codegen to avoid registering anything to the dispatcher (seems harmless, since static dispatch doesn't use the dispatcher).